cip.go

```diff
@@ -149,6 +157,15 @@ func main() {
 		}
 	}
 
+	if len(*manifestBasedSnapshotOf) > 0 {
+		srcRegistry = &reg.RegistryContext{
+			Name:           reg.RegistryName(*manifestBasedSnapshotOf),
+			ServiceAccount: *snapshotSvcAccPtr,
+			Src:            true,
+		}
+	}
```

Comment:
It's surprising (at first glance) that we don't set mfests here, but we do in the `*snapshotPtr` case.  Deliberate?

Reply:
Oops, this is an oversight. Ack